scripts/functional_gem_behavior.sh

```diff
@@ -232,24 +264,14 @@ else
   pass "It does."
 fi
 
-desc "rzo roles with no local .rizzo.json is expected to have no output"
+desc "rzo roles with no personal .rizzo.json is expected to warn"
 stdout=$(mktemp -t XXXXXX.stdout)
 stderr=$(mktemp -t XXXXXX.stderr)
 rzo roles 2> $stderr > $stdout
-if [[ -s $stderr ]]; then
-  echo "STDERR is expected to be empty, got:"
-  cat $stderr
-  fail "rzo roles produced STDERR output."
-else
-  pass "STDERR is zero bytes"
-fi
-if [[ -s $stdout ]]; then
-  echo "STDOUT is expected to be empty, got:"
-  cat $stdout
-  fail "rzo roles produced STDOUT output."
-else
-  pass "STDOUT is zero bytes"
-fi
+
+match "warns about puppetdata not being a directory" 'WARN .*puppetdata is not a directory' $stderr
+match "warns about ghoneycutt-modules not being a directory" 'WARN .*ghoneycutt-modules is not a directory' $stderr
+match "warns about bootstrap not being a directory" 'WARN .*bootstrap is not a directory' $stderr
```

Comment:
after this run mkdir on the three directories and run rzo again. Now no warnings should be present.